### ISSUE TITLE
Twitter Tips and Vimium

### ISSUE THREAD

**Issue by jonathansampson** [brave/brave-browser#5670]
## Description 
Vimium is incapable of finding the Twitter Tip buttons.

## Steps to Reproduce
1. Install [Vimium](https://chrome.google.com/webstore/detail/vimium/dbepggeogbaibhgnhhndojpepiihcmeb?hl=en)
2. Reveal _local hints_ by pressing `f`

## Actual result:
Tip buttons are not given a hint label.

## Expected result:
Tip buttons are given a hint label.

## Reproduces how often: 
Always.

## Brave version (brave://version info)
0.67.125

## Other Additional Information:
Here is a small test to confirm the issue. Note that the container lacking a shadow DOM is properly labeled by Vimium when pressing `f`, while the other is not: https://jsfiddle.net/x6L9n287/

![shadow-container](https://user-images.githubusercontent.com/815158/63086435-7898a980-bf15-11e9-8f87-281abd018252.gif)

**Comment by jonathansampson** [comment]
The move here may be to work with Vimium to better understand the issue, and resolve it from their end.

**Comment by agentofuser** [comment]
Thank you for looking into this! I've also observed this on github tipping buttons, as expected.

**Comment by jonathansampson** [comment]
Closing, as I believe this was from our Muon build.
